migration/src/m0001030_add_original_purl_column.rs

```diff
@@ -0,0 +1,48 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
```

Comment:
@JimFuller-RedHat should i create/drop indexes for the new `original` column in here? If so, can you give me the exact syntax for each, please?

Reply:
if we are doing searches on this column - which include LIKE or partial searches, something like

```
                    DROP INDEX IF EXISTS qualified_purl_original_gist;
                    CREATE INDEX qualified_purl_original_gist ON qualified_purl USING GIST (original gist_trgm_ops);
```

should do it (we cannot represent gist with sea_orm syntax).